libnopegl/src/backends/vk/buffer_vk.c

```diff
@@ -229,19 +230,34 @@ void ngli_buffer_vk_unmap(struct buffer *s)
     vkUnmapMemory(vk->device, s_priv->memory);
 }
 
+void ngli_buffer_vk_ref(struct buffer *s)
+{
+    struct buffer_vk *s_priv = (struct buffer_vk *)s;
+    s_priv->refcount++;
+}
+
+void ngli_buffer_vk_unref(struct buffer *s)
```

Comment:
Without an `unrefp` you will have the issue that the pointer will always remain: the `ngli_freep()` below will only clear the local pointer.

Reply:
Dropped.